SS2-Project/Assets/Starstorm2/Modules/Pickups/Items/Lunar/RelicOfEchelon.cs

```diff
@@ -45,31 +40,32 @@ public sealed class RelicOfEchelon : SS2Item, IContentPackModifier
 
         public static Color echelonColor;
 
-        public override bool IsAvailable(ContentPack contentPack)
+        public override SS2AssetRequest<ItemAssetCollection> AssetRequest<ItemAssetCollection>()
         {
-            return false; //disabled for now
+            return SS2Assets.LoadAssetAsync<ItemAssetCollection>("acRelicOfEchelon", SS2Bundle.Items);
         }
 
-        public override IEnumerator LoadContentAsync()
+        public override void OnAssetCollectionLoaded(AssetCollection assetCollection)
         {
-            //ItemDef - "RelicOfEchelon" - Items
-            //BuffDef - "BuffEchelon" - Indev
-            yield break;
+            //_overlay = assetCollection.FindAsset<Material>("matCognation");
+
+            //I couldn't find a material corresponding to the cognation overlay -Jace
         }
+
+        public override bool IsAvailable(ContentPack contentPack)
+        {
+            return false; //disabled for now
+        }
+
         public override void Initialize()
         {
-            BuffOverlays.AddBuffOverlay(_buffEchelon, _overlay);
+            //BuffOverlays.AddBuffOverlay(SS2Content.Buffs.BuffEchelon, _overlay);
```

Comment:
90% sure SS2Content type fields aren't filled in yet during Initialize. Grab the BuffDef from the asset collection instead.